client/my-sites/hosting/main.js

```diff
@@ -19,27 +19,34 @@ import canSiteViewAtomicHosting from 'state/selectors/can-site-view-atomic-hosti
 import SFTPCard from './sftp-card';
 import PhpMyAdminCard from './phpmyadmin-card';
 import DataLossWarning from './data-loss-warning';
+import PhpVersionCard from './php-version-card';
 import { isEnabled } from 'config';
 
 /**
  * Style dependencies
  */
 import './style.scss';
 
-const Hosting = ( { translate, isDisabled, canViewAtomicHosting } ) => {
+const Hosting = ( { translate, isDisabled, canViewAtomicHosting, siteId } ) => {
 	if ( ! canViewAtomicHosting ) {
 		return null;
 	}
 
-	const sftpPhpMyAdminFeatures = isEnabled( 'hosting/sftp-phpmyadmin' ) ? (
-		<>
-			<div className="hosting__cards">
-				<SFTPCard disabled={ isDisabled } />
-				<PhpMyAdminCard disabled={ isDisabled } />
-			</div>
-			{ ! isDisabled && <DataLossWarning /> }
-		</>
-	) : null;
+	let sftpPhpMyAdminFeatures = null;
+
+	// This is also enforced on the server, please remove client checks later.
+	// ID of site added 31 Oct 2019, so only sites newer currently eligible
+	if ( isEnabled( 'hosting/sftp-phpmyadmin' ) && siteId > 168768859 ) {
```

Comment:
Moved the check here, as we want to enable hosting panel when we enable the PHP version switcher.